### PR TITLE
SEP-8: add an optional message field along with next_url

### PR DESCRIPTION
**Pull request by howardtw**
**What**

This PR adds an optional field `message` to the `follow_next_url` response.

**Why**

The client can choose to display the message to the user if the approval server requires further actions.

### CODE REVIEW

File: ecosystem/sep-0008.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ Name | Type | Description
 ```json
 {
   "status": "action_required",
-  "message": "Please sign the terms of service via the provided URL.",
+  "message": "Please provide the information specified in the action fields via the provided URL.",
```
Comment (leighmcculloch) [inline ecosystem/sep-0008.md:244]: ❗ The message field is intended to communicate to the end user, but the new message in this PR sounds like the audience is the client/application developer. An end user won't know what `action fields` are, or necessarily know anything about `URL`s.

```suggestion
  "message": "Please provide an email address and mobile phone number.",
```

File: ecosystem/sep-0008.md [hunk 2]
```diff
@@ -298,13 +298,15 @@ There are two possible values for `action_method`.
...
+   `message` | string | (optional) A human readable string containing information regarding the further action required.
 
    Response Example:
 
    ```json
    {
      "result": "follow_next_url",
-     "next_url": "https://..."
+     "next_url": "https://...",
+     "message": "Please sign the terms of service via the provided URL."
```
Comment (leighmcculloch) [inline ecosystem/sep-0008.md:309]: ❗ Same as my other comment regarding audience:
```suggestion
     "message": "Please sign the terms of service."
```

**Review by leighmcculloch — COMMENTED** [review]
Two comments (❗) below I think need addressing.

File: ecosystem/sep-0008.md (hunk 1 above)
Comment (howardtw) [reply ecosystem/sep-0008.md]: Good point. https://github.com/stellar/stellar-protocol/pull/859/commits/d251e4a86f38d7301e31e55358ae2af6e23b4815

File: ecosystem/sep-0008.md (hunk 2 above)
Comment (howardtw) [reply ecosystem/sep-0008.md]: Addressed in the same commit: https://github.com/stellar/stellar-protocol/pull/859/commits/d251e4a86f38d7301e31e55358ae2af6e23b4815

**Review by leighmcculloch — APPROVED** [review]

**Pull request merged**